library/globals.inc.php

```diff
@@ -1320,7 +1320,7 @@
          '6' => xl('Saturday')
        ),
        '1',
```

Comment:
This is a good idea. There is a quick listing of the constants here that are easy to search:
contrib/util/language_translations/currentConstants.txt
https://github.com/openemr/openemr/blob/master/contrib/util/language_translations/currentConstants.txt